lib/src/layer/tile_layer/tile.dart

```diff
@@ -4,6 +4,7 @@ import 'package:flutter/widgets.dart';
 import 'package:flutter_map/flutter_map.dart';
 
 /// The widget for a single tile used for the [TileLayer].
+/// If proceed with the implementation of the CustomPainter and the TileModel this Widget should be removed.
```

Comment:
We should deprecate this.

Reply:
I deprecated it initially, but tests didn't pass anymore, as class `_TileState extends State<Tile>`  was using deprecated `Tile` class. So I am not sure how to manage this? Just delete class altogether?

Reply:
Deprecation is transitive. Mark both classes as deprecated, and it should work. However, just don't worry about this for now, and I'll do it at the end to make sure it's correct.

Reply:
`Tile` is not exported publically so it should just get removed with this pr. It got completely replaced it with `TileModel`